core/src/main/java/org/apache/iceberg/avro/AvroSchemaUtil.java

```diff
@@ -90,6 +90,25 @@ static boolean hasIds(Schema schema) {
     return AvroCustomOrderSchemaVisitor.visit(schema, new HasIds());
   }
 
+  /**
+   * Check if any of the nodes in a given avro schema is missing an ID
+   * <p>
+   * To have an ID recognizable by Iceberg core API:
```

Comment:
This still references the "Iceberg core API"

Reply:
Ack, removed.